tools/benchmark.py

```diff
@@ -62,6 +63,16 @@ def run_thread_count_benchmark(deno_path):
     return thread_count_map
 
 
+def run_syscall_count_benchmark(deno_path):
+    syscall_count_map = {}
+    syscall_count_map["hello"] = count_strace_syscall(
+        None,
+        lambda line: "<... unfinished>" not in line and  # <unfinished> and <resume> repeats same syscall
+        "+++ exited with" not in line,  # lines labelling exit
+        [deno_path, "tests/002_hello.ts", "--reload"])
+    return syscall_count_map
```

Comment:
Since the parsing of strace is becoming more complex, I think you should store a trace
```
strace deno tests/001_hello.js >& tools/testdata/strace_001.txt
```
And then check that you get the correct value in benchmark_test.py. This will be much more robust and debuggable than just testing `> 1`.

Reply:
Just noticed that `strace` has an option `-c` that generates a nice call summary. However, summaries show the same program calling syscalls different times and with slightly different orders. For example, in my testing, `futex` is called with either 3 or 4 times and varies across runs. Dunno if same program would even have a slightly different set of syscalls running each time

Reply:
The syscalls are definitely not going to be constant - there's a lot of complexity there. Using the `-c` option sounds great - store the output into `//tools/testdata/strace_summary.txt` and check that you can parse it independently of the benchmark.

Reply:
Just want to clarify, the purpose of this file is to ensure that we can do parsing correctly, right? For example, if we have a table that looks like
```
calls     name
==========
4           futex
2           mmap
```
then would validate in our benchmark test that the our parsing logic would give us exactly 6 for total syscall count. Am I understanding correctly?

Reply:
Yes
```python
let r = strace_parse("tools/testdata/systrace_count.txt");
assert r['futex'] == 4
assert r['mmap'] == 2
```